run.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash -e
+
+SRC="/app/extra/wt.tar.gz"
+
+# Extract
+tar -xv --gzip -f $SRC -C $XDG_DATA_HOME
```

Comment:
This ensures that the script doesn't extract at runtime if the `WarThunder` directory exists.
```suggestion
if [[ ! -d "${XDG_DATA_HOME}/WarThunder" ]]; then
	tar -xv --gzip -f $SRC -C $XDG_DATA_HOME
fi
```

Reply:
Negative, This must exist because the launcher itself could be updated.

Reply:
Oh yikes.

Reply:
@Doomsdayrs try with this patch. I just simulated on my end and it works. You'll notice `"$XDG_DATA_HOME"/sha_check`, a file that holds the current archive's hash. The new script checks the archive's hash, compares with the hash in the file and extracts if they're different.

So it should work with updates now while avoiding redundantly extracting every time the user launches.